server.js

```diff
@@ -18,8 +18,25 @@ const packageJson = require('./package.json')
 const routes = require('./app/routes.js')
 const utils = require('./lib/utils.js')
 
+var useV6 = false
```

Comment:
It might be good to add a comment about what "v6" is here, or maybe just link to the changelog entry.